spring-grpc-spring-boot-autoconfigure/src/test/java/org/springframework/grpc/autoconfigure/server/GrpcServletAutoConfigurationTests.java

```diff
@@ -90,7 +74,40 @@ void whenNoBindableServicesRegisteredAutoConfigurationIsSkipped() {
 
 	@Test
 	void whenWebApplicationServletIsAutoConfigured() {
-		this.contextRunner().run((context) -> assertThat(context).getBean(ServletRegistrationBean.class).isNotNull());
+		this.contextRunner().run((context) -> {
+			assertThat(context).getBean(ServletRegistrationBean.class)
+				.isNotNull()
+				.extracting("servlet")
+				.isInstanceOf(GrpcServlet.class)
+				.extracting("servletAdapter")
+				.isInstanceOf(ServletAdapter.class)
+				.extracting("maxInboundMessageSize")
+				.isEqualTo(GrpcUtil.DEFAULT_MAX_MESSAGE_SIZE);
+		});
+	}
+
+	@Test
+	void whenCustomizerIsRegistered() {
```

Comment:
I think we can simplify this just a bit w/ a mock - something like:
```java
	@Test
	void whenCustomizerIsRegistered() {
		ServerBuilderCustomizer<ServletServerBuilder> customizer = mock();
		this.contextRunner()
			.withBean(ServerBuilderCustomizer.class, () -> customizer)
			.run(context -> verify(customizer).customize(any(ServletServerBuilder.class)));
	}
```

Wdyt?

Reply:
make sense. fixed in 2f50e5c27732722dd6027127135d1673f1503a59